### ISSUE TITLE
Allow updating an existing api when importing via import/openapi.

### ISSUE THREAD

**Issue by togiles** [a85/POSTMan-Chrome-Extension#10949]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

We have built automation around publishing APIs via the https://api.getpostman.com/import/openapi endpoint, however the existing APIs only allow importing new APIs and do not support updating an existing API when importing a newer version.

### Describe the solution you'd like

We would like to be able to have the ability to import the same API over and over without creating new collections on each import, but instead replacing/updating the existing API.

In short, we'd like for https://api.getpostman.com/import/openapi to be idempotent.

### Describe alternatives you've considered

We are building a solution to first delete the collection and then re-import, but this can be error prone and requires extra book-keeping on our side we'd prefer to offload to postman :).

**Comment by pierre-wehbe** [comment]
+10000 :)

**Comment by engineoss** [comment]
We also would like to see this feature!

**Comment by jsaguiar** [comment]
This will push me and my team to look outside Postman Pro

**Comment by preethammavin** [comment]
Hey, we have released SpecHub and also a newer version of Postman API to support this workflow. You can now import your API Spec into [SpecHub](https://learning.postman.com/docs/design-apis/specifications/overview/), then generate a collection from it once. 

Once this is setup in a workspace, you can update the Spec & Collections with newer versions of the Spec using Postman API. 

The Postman API to achieve this can be found [here](https://www.postman.com/postman/postman-public-workspace/folder/4qpsuuz/specs), and then also update the collection from Postman API continuously.
